distribution/engine/THIRD-PARTY/NOTICE

```diff
@@ -346,11 +376,31 @@ The license information can be found along with the copyright notices.
 Copyright notices related to this dependency can be found in the directory `org.graalvm.shadowed.icu4j-23.1.0`.
 
 
+'json', licensed under the Universal Permissive License, Version 1.0, is distributed with the engine.
+The license file can be found at `licenses/Universal_Permissive_License__Version_1.0`.
+Copyright notices related to this dependency can be found in the directory `org.graalvm.shadowed.json-23.1.0`.
+
+
+'profiler-tool', licensed under the GNU General Public License, version 2, with the Classpath Exception, is distributed with the engine.
+The license file can be found at `licenses/GNU_General_Public_License__version_2__with_the_Classpath_Exception`.
+Copyright notices related to this dependency can be found in the directory `org.graalvm.tools.profiler-tool-23.1.0`.
```

Comment:
I have no idea if the way we distribute the `profiler-tool` satisfies the requirements of the GNU GPL license with classpath exception.

If I understand correctly, we are distributing these files inside of our JAR, right? Or is it a separate JAR? 

1. We need to clarify how this dependency is distributed.
2. Then check if the way we do this is OK with the license - we should consult someone who has more legal knowledge about this.

Reply:
I had been in a [legal battle about GPL+CP Exception with ASF](http://wiki.apidesign.org/wiki/DirectAction) and I won. That makes me believe I am qualified...

>>    If I understand correctly, we are distributing these files inside of our JAR, right? Or is it a separate JAR?
>
> It is a separate JAR. We basically take those Jars from Maven central 
> and just put it inside the component directory in our distribution, without any modifications.

Good. Distributing GPL+CPEx JARs is certainly OK and we can even claim the whole thing is under Apache License. Radek is right that repackaging might be tricker - good that we are not doing it.